src/util/parseDocument.ts

```diff
@@ -86,16 +87,33 @@ const getStyleAttribute = (element: JSXElement): IStyleAttribute | null => {
   }
 
   // Filter and transform properties
-  const properties = (styleAttr.value.expression.properties.filter(
-    p =>
-      p.type === "ObjectProperty" && supportedValueTypes.includes(p.value.type)
-  ) as ObjectProperty[]).map(p => ({
-    key: (p.key as Identifier).name as string,
-    value:
-      p.value.type === "TemplateLiteral"
-        ? p.value.quasis.map(el => el.value.raw).join("")
-        : (p.value as StringLiteral).value,
-  }));
+  const properties = (
+    styleAttr.value.expression.properties.filter(
+      p =>
+        p.type === "ObjectProperty" &&
+        supportedValueTypes.includes(p.value.type)
+    ) as ObjectProperty[]
```

Comment:
You definetly improved the formatting here :) While we are at it, could you rename `p` to `prop` or `property`?

Reply:
sure! i've pushed up a commit. the formatting improvements were simply a result of me running [prettier](https://prettier.io/) on this file